doc/reference/examples.rst

```diff
@@ -3,7 +3,7 @@
 Examples
 ========
 
-Full examples of Plutus Applications can be found in the ``plutus`` `repository <https://github.com/input-output-hk/plutus/tree/master/plutus-use-cases>`_.
+Full examples of Plutus Applications can be found in the ``plutus`` `repository <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-use-cases>`_.
```

Comment:
The text is also wrong, should say "plutus-apps repository"

Reply:
https://github.com/input-output-hk/plutus-apps/pull/227